regression/contracts/assigns_enforce_18/test.desc

```diff
@@ -0,0 +1,21 @@
+CORE
+main.c
+--enforce-all-contracts
+^EXIT=10$
+^SIGNAL=0$
+^\[foo.\d+\] line \d+ Check that \*xp is assignable: SUCCESS$
+^\[foo.\d+\] line \d+ Check that \*bar is assignable: SUCCESS$
+^\[foo.\d+\] line \d+ Check that \*baz is assignable: FAILURE$
+^\[foo.\d+\] line \d+ Check that \*quz is assignable: FAILURE$
+^VERIFICATION FAILED$
+--
+--
+We had problems before when freeing a variable, but still kepting it on
+the writable set, which lead to dead variables issues. Anything that is local,
+including local allocations are now tracked by our freely assignable set;
+therefore, CBMC doesn't include these local variables in its alias check
+anymore (which avoids the issue previously described). Assignments to
+anything in our freely assignable set are bypassed. It’s important to
+notice that we track the address of all objects, thus, if a local pointer
+is created and points to something that is not local (e.g., inputs or global
+variables), CBMC will check every assignment to it.
```

Comment:
"still kepting it on the writable set, which lead to dead variables issues." -> "still keeping it in the writable set, which lead to dead variable issues."

The information in this test appears to be useful to know, but in the wrong place. This describes how the implementation works, not what the test is actually checking for. I expect that someone trying to understand the code for contracts would look at the source code as a place to find information on how it works.